### ISSUE TITLE
What is the logits?

### ISSUE THREAD

**Issue by windhxs** [timoschick/pet#57]
In class `InputExample`, it has a property `logits`, I am confused of this property, what does it used for ?
